### PR TITLE
New configurables for TPC

### PR DESCRIPTION
**Pull request by marslandALICE**

### CODE REVIEW

**Comment by benedikt-voelkel** [comment]
Hej @marslandALICE 
Do you consider this PR ready for review? In that case, could you please resolve the conflicts?
Otherwise, in case you are still working on it, could you change it to a draft PR?
Thanks a lot

**Comment by marslandALICE** [comment]
Hi Benedict, It should be ok now.

File: MC/bin/o2dpg_sim_workflow.py [hunk 1]
```diff
@@ -740,7 +740,7 @@ def putConfigValuesNew(listOfMainKeys=[], localCF = {}):
    TPCDigitask=createTask(name='tpcdigi_'+str(tf), needs=tpcdigineeds,
                           tf=tf, cwd=timeframeworkdir, lab=["DIGI"], cpu=NWORKERS, mem='9000')
    TPCDigitask['cmd'] = ('','ln -nfs ../bkg_HitsTPC.root . ;')[doembedding]
-   TPCDigitask['cmd'] += '${O2_ROOT}/bin/o2-sim-digitizer-workflow ' + getDPL_global_options() + ' -n ' + str(args.ns) + simsoption + ' --onlyDet TPC --TPCuseCCDB --interactionRate ' + str(INTRATE) + '  --tpc-lanes ' + str(NWORKERS) + ' --incontext ' + str(CONTEXTFILE) + ' --disable-write-ini ' + putConfigValuesNew(["TPCGasParam","TPCGEMParam","TPCEleParam","TPCITCorr"],localCF={"DigiParams.maxOrbitsToDigitize" : str(orbitsPerTF)})
+   TPCDigitask['cmd'] += '${O2_ROOT}/bin/o2-sim-digitizer-workflow ' + getDPL_global_options() + ' -n ' + str(args.ns) + simsoption + ' --onlyDet TPC --TPCuseCCDB --interactionRate ' + str(INTRATE) + '  --tpc-lanes ' + str(NWORKERS) + ' --incontext ' + str(CONTEXTFILE) + ' --disable-write-ini ' + putConfigValuesNew(["TPCGasParam","TPCGEMParam","TPCEleParam","TPCITCorr","TPCDetParam"],localCF={"DigiParams.maxOrbitsToDigitize" : str(orbitsPerTF)})
```
Comment (benedikt-voelkel) [inline MC/bin/o2dpg_sim_workflow.py:743]: Where do you get the `TPCDetParams` from? By default, they are not there in fact(see https://github.com/AliceO2Group/O2DPG/blob/master/MC/bin/o2dpg_sim_config.py). Only for the anchoring they might be.

**Review by benedikt-voelkel — APPROVED** [review]
As discussed with @sawenzel

**Pull request merged**